### ISSUE TITLE
decode wallet.aes.json

### ISSUE THREAD

**Issue by winteraz** [gurnec/btcrecover#197]
Is it possible to decode wallet.aes.json using btcrecover? I know the password so no "trial and error" is required.

**Comment by pedrohrcunha** [comment]
Yes, you can. Please run this program: https://github.com/gurnec/btcrecover/blob/master/extract-scripts/download-blockchain-wallet.py

Do you know how to do it?

**Comment by travelmassive** [comment]
Not that I'm aware of (@pedrohrcunha I think that @winteraz want's to decode the wallet into plaintext, not download it).

From what I can see, the program decodes chunks (to be more efficient and speedy) so you'd have to investigate the code base a little to determine how to print out the decrypted wallet.

I'm hoping that there is a tool or command line that @gurnec has developed that I've overlooked :)

If you are proficient with compiling Java and using command line (sorry, no time to assist if you don't) you could look at compiling the tool I've used.

Here's the gist (you will need the jar files also - I could email them to you).

https://gist.github.com/travelmassive/e6d060cb12a723e62290b1bedfe96775

**Comment by anhday22** [comment]
Currently I have new technology to help open my wallet with a high success rate. If any of your wallet hasn't been successfully opened, please send it to me. I will try to open them.
http://recoverywalletcrypto.com
www.fb.me/recoverywallet
Thank

**Comment by nori19870522** [comment]
I am Japanese and do password analysis.
https://keychainx.io/ 

This is Robert's article.
https://news.bitstarz.com/bitstarz-news-interview-robert-rhodin-keychainx-ceo  

I may be able to find the password you lost. Do you need help with your login?

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover

**Comment by viladmir2022** [comment]
hi, if you still have wallet problem. I can recover your lost wallet.
I have powerful servers and excellent software that can find any password.
and my fee is only 15-25% of the recovered fund
here is my telegram id: @viladmir2022  and Gmail: walletsrecoveryservice@gmail.com
